config/environments/production.rb

```diff
@@ -71,11 +70,11 @@
   # config.action_mailer.raise_delivery_errors = false
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
-  # English when a translation cannot be found).
-  config.i18n.fallbacks = [:en]
+  # the I18n.default_locale when a translation cannot be found).
+  config.i18n.fallbacks = true
```

Comment:
Since we set default to en elsewhere, this is effectively not a change -- but seems like a nice to have on an instance where the default was NOT set to en?

Reply:
The source language is English, so the fallback should be as well. See https://github.com/mastodon/mastodon/pull/24727

Reply:
Ah, did not consider the "instance has default locale set but that locale is incomplete scenario". Will update this and maybe add comment explaining so future-me does not do this again.